test/Cli/Func.E2E.Tests/Commands/FuncInit/GenericValidationInitTests.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Azure.Functions.Cli.E2E.Tests.Traits;
+using Azure.Functions.Cli.TestFramework.Assertions;
+using Azure.Functions.Cli.TestFramework.Commands;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Azure.Functions.Cli.E2E.Tests.Commands.FuncInit
+{
+    public class GenericValidationInitTests(ITestOutputHelper log) : BaseE2ETests(log)
+    {
+        [Fact]
+        public void Init_Function_App_With_unknown_worker_runtime()
```

Comment:
```suggestion
        public void Init_WithUnknownWorkerRuntime_DisplayNotValidOptionError()
```

remember this format: Command_Action_Result